docs/configurations/_index.md

```diff
@@ -0,0 +1,213 @@
+
+### Spark Runtime Write Options
+
+| Spark option           | Default                    | Description                                                  |
+| ---------------------- | -------------------------- | ------------------------------------------------------------ |
+| write-format           | Table write.format.default | File format to use for this write operation; parquet, avro, or orc |
+| target-file-size-bytes | As per table property      | Overrides this table's write.target-file-size-bytes          |
+| check-nullability      | true                       | Sets the nullable check on fields                            |
+| snapshot-property._custom-key_    | null            | Adds an entry with custom-key and corresponding value in the snapshot summary  |
+| fanout-enabled       | false        | Overrides this table's write.spark.fanout.enabled  |
+| check-ordering       | true        | Checks if input schema and table schema are same  |
+| isolation-level | null | Desired isolation level for Dataframe overwrite operations.  `null` => no checks (for idempotent writes), `serializable` => check for concurrent inserts or deletes in destination partitions, `snapshot` => checks for concurrent deletes in destination partitions. |
+| validate-from-snapshot-id | null | If isolation level is set, id of base snapshot from which to check concurrent write conflicts into a table. Should be the snapshot before any reads from the table. Can be obtained via [Table API](../../api#table-metadata) or [Snapshots table](../spark-queries#snapshots). If null, the table's oldest known snapshot is used. |
+
+### Hadoop Configurations
+
+The following properties from the Hadoop configuration are used by the Hive Metastore connector.
+
+| Property                              | Default          | Description                                                                        |
+| ------------------------------------- | ---------------- | ---------------------------------------------------------------------------------- |
+| iceberg.hive.client-pool-size         | 5                | The size of the Hive client pool when tracking tables in HMS                       |
+| iceberg.hive.lock-timeout-ms          | 180000 (3 min)   | Maximum time in milliseconds to acquire a lock                                     |
+| iceberg.hive.lock-check-min-wait-ms   | 50               | Minimum time in milliseconds to check back on the status of lock acquisition       |
+| iceberg.hive.lock-check-max-wait-ms   | 5000             | Maximum time in milliseconds to check back on the status of lock acquisition       |
+| external.table.purge        | true                       | If all data and metadata should be purged in a table by default when the table is dropped |
+| gc.enabled                  | true                       | if all data and metadata should be purged in the table by default |
```

Comment:
Do we want to add some comment about that these properties are synchronized? In the Hive doc we have this:
```
Changing gc.enabled on the Iceberg table via UpdateProperties, updates
external.table.purge on the HMS table accordingly. Setting external.table.purge as a table
prop during Hive CREATE TABLE pushes gc.enabled down to the Iceberg table properties.
This ensures these properties are always consistent at table level between Hive and
Iceberg.
```

Reply:
Also the `external.table.purge` and `gc.enabled` are table level properties in the HMS, and not global config properties like the other hadoop properties.

Also somewhere it would be good to state that the HMS table properties and the Iceberg table properties are kept in sync for HiveCatalog tables i Hive 4.0.0. So if some change is done in the properties with a Hive query, or any other external tool, then both the Iceberg table properties and the HMS properties are changed. This is sadly not true for older Hive versions, where the changes done by the Hive queries are not propagated to the Iceberg tables, the sync is done only from the other direction

Reply:
Updated this to move those properties to a `Hive Table Level Configurations` section and include the notes you've mentioned.

Reply:
Thanks @samredai